src/resources/help/en_US/relnotes3.2.0.html

```diff
@@ -90,7 +90,7 @@
 <a name="Bugs_fixed"></a>
 <p><b><font size="4">Other bugs fixed in version 3.2</font></b></p>
 <ul>
-  <li>Bug Fix 1</li>
+  <li>Expression checkbox indicates display database AND auto-generated expressions.</li>
```

Comment:
I would make this a bit more descriptive. Something along the lines of "The checkbox in the Expression window that turns on the display of expressions from the database now indicates that it displays expressions from the database and auto generated ones."